common/tools/dev-tool/src/util/customization/interfaces.ts

```diff
@@ -68,3 +78,67 @@ export function removeProperties(
     }
   }
 }
+
+function findCallSignature(
+  interfaceDeclaration: InterfaceDeclaration,
+  callSignature: CallSignatureDeclaration
+): CallSignatureDeclaration | undefined {
+  function typeEquals(a: Type, b: Type) {
+    // Need to handle cases where the type is imported
+    const aStr = a?.getText()?.replace(/import\(\".+\"\)\./, "");
+    const bStr = b?.getText()?.replace(/import\(\".+\"\)\./, "");
+    return aStr && bStr && aStr === bStr;
+  }
+  return interfaceDeclaration.getCallSignature((signature) => {
+    if (signature.getParameters().length !== callSignature.getParameters().length) {
+      return false;
+    }
+    signature.getReturnTypeNode;
```

Comment:
What is the purpose of this line?

Reply:
removed, it was just a leftover from a refactor.

Reply:
Thanks, did you open a new PR? This one is already merged without this change.